concepticondata/references/references.bib

```diff
@@ -4266,5 +4266,12 @@ @Article{Nordenskioeld1905
   Number                              = {3},
   Journal                             = {Ymer},
   Title                               = {Beiträge zur Kenntnis einiger Indianerstämme des Rio Madre de Dios-gebietes [Contributions to the Knowledge about some Indian tribes of the area of the Rio Madre de Dios]},
-  Year                                = {1905},
+  Year                                = {1905}
+}
+
+@Article{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
+  Title                               = {West Old Turkic},
+  Publisher                           = {Harrassowitz},
+  Year                                = {2011}
```

Comment:
The bibliography misses information on address (Wiesbaden) and the subtitle (Turkic loanwords in Hungarian). Could you add those?

Reply:
Oh yes, the most important change: The bibtype is `Book`, not `Article`. Thanks for pointing that out.

Reply:
Implemented the changes with d8aeaad31eb06b7c9165d9202b5c5c5fa1bc2744

Reply:
> Implemented the changes with [d8aeaad](https://github.com/concepticon/concepticon-data/commit/d8aeaad31eb06b7c9165d9202b5c5c5fa1bc2744)

We see that the changes are implemented with the note "outdated", so you do not need to add the comment above. You can use the "Resolve conversation" button so that the comments are collapsed. I'll make a final check at the end of the review to see if all changes are correctly implemented.